website/projects/spoke/src/app/core/state/page.state.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
+import { DataAction, StateRepository } from '@ngxs-labs/data/decorators';
+import { NgxsImmutableDataRepository } from '@ngxs-labs/data/repositories';
+import { State } from '@ngxs/store';
+
+interface PageStateModel {
+  allowTelemetry: boolean;
+}
+
+const LOCAL_STORAGE_ALLOW_TELEMETRY_KEY = 'ALLOW_TELEMETRY';
+
+@StateRepository()
+@State<PageStateModel>({
+  name: 'page',
+  defaults: {
+    allowTelemetry: false
+  }
+})
+@Injectable()
+export class PageState extends NgxsImmutableDataRepository<PageStateModel> {
+
```

Comment:
Add ngxsOniInit similar to https://github.com/cns-iu/a2agc-dataset/blob/develop/website/src/app/core/state/page/page.state.ts#L28 to initialize allowTelemetry correctly.